src/ccmain/tesseractclass.h

```diff
@@ -1001,6 +1004,7 @@ class TESS_API Tesseract : public Wordrec {
   int scaled_factor_;
   FCOORD deskew_;
   FCOORD reskew_;
+  float gradient_;
```

Comment:
The new member variable should be initialized in the constructor, too.

Reply:
Edited.